roles/inspec/tasks/main.yml

```diff
@@ -65,14 +74,16 @@
     src: etc/inspec/host-controls/controls/openstack-security.rb
     dest: /etc/inspec/host-controls/controls/
     mode: 0644
-  when: inventory_hostname in groups['controller'] or  inventory_hostname in groups['compute'] or inventory_hostname in groups['cinder_volume']|default([])
+  when: inventory_hostname in groups['controller'] or
+        inventory_hostname in groups['compute'] or
+        inventory_hostname in groups['cinder_volume']|default([])
 
 - name: rhel control files
   template:
     src: etc/inspec/host-controls/controls/rhel7-stigs.rb
     dest: /etc/inspec/host-controls/controls/
     mode: 0644
-  when: ursula_os == 'rhel'
+  when: inspec.controls.rhel7.enable
```

Comment:
always best to use `|bool` when getting boolean from vars to ensure its evaluated as a boolean not a string.